### PR TITLE
Initial mockup of the Reptilian species

### PR DESCRIPTION
**Pull request by metalgearsloth**
This https://github.com/space-wizards/space-station-14/pull/6099

I only got tests working but it still needs a review because I smell code issues.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Species/reptilian.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
+- type: entity
+  save: false
+  name: Urist McHands The Lizardperson
+  parent: MobHumanBase
+  id: MobReptilianBase
+  abstract: true
+  description: A miserable pile of scales.
+  components:
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/Species/reptilian.yml:8]: I merged in master but the reptilian mob still had doorbumpopener component which tells me there needs to be more shared with humans.

File: Content.Client/Lobby/UI/LobbyCharacterPreviewPanel.cs [hunk 2]
```diff
@@ -27,6 +27,7 @@ public sealed class LobbyCharacterPreviewPanel : Control
         private readonly IClientPreferencesManager _preferencesManager;
         private readonly IPrototypeManager _prototypeManager;
         private EntityUid? _previewDummy;
+        private string? _previewDummySpecies;
```
Comment (moonheart08) [inline Content.Client/Lobby/UI/LobbyCharacterPreviewPanel.cs:30]: This isn't used anywhere.

File: Resources/Prototypes/Entities/Mobs/Species/reptilian.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Mobs/Species/reptilian.yml:8]: Same with innate actions being duplicated across reptile.

Comment (moonheart08) [reply Resources/Prototypes/Entities/Mobs/Species/reptilian.yml:8]: As is, we can't automatically reuse/merge the sprite layers due to how prototypes work, so I don't think anything more CAN be shared.

**Review by moonheart08 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/Species/reptilian.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Mobs/Species/reptilian.yml:8]: Doesn't the entire sprite get bulldozed? That seems okay to me.

Master shouldn't have to care about manually ensuring that reptiles and humans are copy-pasted exactly.

Comment (moonheart08) [reply Resources/Prototypes/Entities/Mobs/Species/reptilian.yml:8]: > Doesn't the entire sprite get bulldozed? That seems okay to me.
> 
> Master shouldn't have to care about manually ensuring that reptiles and humans are copy-pasted exactly.

I'll note we already have this issue with the other species, too. It's something that'll be solved when composition is in.

**Comment by metalgearsloth** [comment]
DNM until someone comes up with interesting gameplay mechanics.

**Comment by Mith-randalf** [comment]
Here's my current 'throw shit at the wall to see what sticks' suggestions for lizard mechanics:
https://hackmd.io/@Mithrandalf/rJVFtopgc

IDK if any of these suggestions are any good, but it is offered humbly.

**Comment by metalgearsloth** [comment]
Gonna turn off roundstart for now and get it merged so admins can at least RP.

**Pull request merged**